playwright-java/src/test/java/com/browserstack/PlaywrightParallelTest.java

```diff
@@ -0,0 +1,123 @@
+        deviceOne.executeTest(jsonObject);
+    }
+}
+
+class DeviceTwo implements Runnable {
+    public void run() {
+        JsonObject jsonObject = new JsonObject();
+        jsonObject.addProperty("browser", "edge");  // allowed browsers are `chrome`, `edge`, `playwright-chromium`, `playwright-firefox` and `playwright-webkit`
+        jsonObject.addProperty("browser_version", "latest");
+        jsonObject.addProperty("os", "osx");
+        jsonObject.addProperty("os_version", "catalina");
+        jsonObject.addProperty("name", "Branded Microsoft Edge on Catalina");
+        jsonObject.addProperty("build", "playwright-build-2");
+        PlaywrightParallelTest deviceTwo = new PlaywrightParallelTest();
+        deviceTwo.executeTest(jsonObject);
+    }
+}
+
+class DeviceThree implements Runnable {
+    public void run() {
+        JsonObject jsonObject = new JsonObject();
+        jsonObject.addProperty("browser", "playwright-firefox");    // allowed browsers are `chrome`, `edge`, `playwright-chromium`, `playwright-firefox` and `playwright-webkit`
+        jsonObject.addProperty("os", "osx");
+        jsonObject.addProperty("os_version", "catalina");
+        jsonObject.addProperty("name", "Playwright firefox on Catalina");
+        jsonObject.addProperty("build", "playwright-build-2");
+        PlaywrightParallelTest deviceThree = new PlaywrightParallelTest();
+        deviceThree.executeTest(jsonObject);
+    }
+}
+
+class DeviceFour implements Runnable {
+    public void run() {
+        JsonObject jsonObject = new JsonObject();
+        jsonObject.addProperty("browser", "playwright-webkit"); // allowed browsers are `chrome`, `edge`, `playwright-chromium`, `playwright-firefox` and `playwright-webkit`
+        jsonObject.addProperty("os", "osx");
+        jsonObject.addProperty("os_version", "catalina");
+        jsonObject.addProperty("name", "Playwright webkit on Catalina");
+        jsonObject.addProperty("build", "playwright-build-2");
+        PlaywrightParallelTest deviceFour = new PlaywrightParallelTest();
+        deviceFour.executeTest(jsonObject);
+    }
+}
+
+class DeviceFive implements Runnable {
+    public void run() {
+        JsonObject jsonObject = new JsonObject();
+        jsonObject.addProperty("browser", "playwright-chromium");   // allowed browsers are `chrome`, `edge`, `playwright-chromium`, `playwright-firefox` and `playwright-webkit`
+        jsonObject.addProperty("os", "osx");
+        jsonObject.addProperty("os_version", "catalina");
+        jsonObject.addProperty("name", "Chrome on Win10");
+        jsonObject.addProperty("build", "playwright-build-2");
+        PlaywrightParallelTest deviceFive = new PlaywrightParallelTest();
+        deviceFive.executeTest(jsonObject);
+    }
+}
+
+public class PlaywrightParallelTest {
+
+    public static void main(String[] args) throws Exception {
+        Thread threadOne = new Thread(new DeviceOne());
```

Comment:
Can we make an array of combinations and iterate over the combinations and start new threads with it. Also, after all threads have started, we should make thread.join() and wait till all the threads gets completed.